scripts_wip/WIN_Manage_NextDNS_App.ps1

```diff
@@ -0,0 +1,77 @@
+<#
+Requires client or site variable for nextdns as "NextDNS" and action as either install, uninstall, start, stop, restart or upgrade (install is the default)
+NextDNS is the ID of the NextDNS Configuration obtained from https://my.nextdns.io/
+This script will also install the NextDNS root certificate by default, to allow for a pretty block page
+#>
```

Comment:
Add your syntax in the header comments so when we move it live it's already there to copy/paste :)

`-nextdns {{site.NextDNS}} -action {(install) | uninstall | start | stop | restart | upgrade}`